src/_includes/layout.njk

```diff
@@ -147,22 +147,22 @@
           <div class="col-md-6">
             <ul class="list-inline social-buttons">
               <li class="list-inline-item">
-                <a href="https://www.facebook.com/codeconnector" target="blank">
+                <a href="https://www.facebook.com/codeconnector" target="blank" aria-label="Code Connector's Facebook navigation link">
                   <i class="fab fa-facebook-f"></i>
                 </a>
-              </li>
+              </li> 
               <li class="list-inline-item">
-                <a href="https://twitter.com/codeconnector_" target="blank">
-                  <i class="fab fa-twitter"></i>
+                <a href="https://twitter.com/codeconnector_" target="blank" aria-label="Code Connector's Twitter navigation link">
+                  <i class="fab fa-twitter" title="Navigate to Twitter"></i>
```

Comment:
What makes you add a title attribute? I am not sure it is needed.

Reply:
My thought process was any **link** with an **Icon** needed a title. I thought that was the Accessibility issue. Was that incorrect.

Reply:
I'm actually not sure, but I thought that when you do an aria-label on something, it labels the whole thing, so whatever is inside it is ignored, so the title would be ignored. We can leave it or if you want to play with accessibility testing, we can try with and without the title to see if there is a difference.